libs/interpret/src/lib/MLIDashboard/Controls/WhatIfTab/WhatIfTab.tsx

```diff
@@ -403,6 +404,7 @@ export class WhatIfTab extends React.PureComponent<
                     canDither={
                       this.state.chartProps.chartType === ChartTypes.Scatter
                     }
+                    allowTreatAsCategorical
```

Comment:
What does the new variable ''allowTreatAsCategorical" do? Looks like it is used in a lot of places.

Reply:
Discussed offline. allowTreatAsCategorical is new param to allow that toggle or not. As we decided for big data we dont allow treat as cat', I need to add an extra param to handle this case. For big data allowTreatAsCategorical is sent as false. allowTreatAsCategorical is true we show toggle .